tests/sympc/protocol/protocol_test.py

```diff
@@ -23,8 +24,27 @@ def test_register_protocol() -> None:
 def test_exception_no_share_class() -> None:
     with pytest.raises(ValueError):
 
-        class TestProtocolException(metaclass=Protocol):
-            ...
+        class TestProtocolShareException(metaclass=Protocol):
+            security_levels = ["semi-honest"]
+
+
+def test_exception_no_security_levels() -> None:
+    with pytest.raises(ValueError):
+
+        class TestProtocolSecurityException(metaclass=Protocol):
```

Comment:
What exactly is this class?

Reply:
In Protocol metaclass I raise an error when , security_level attribute is not defined in a protocol.The `TestProtocolSecurityException` is a test case for that scenario.

Reply:
Oh that's nice.